File: src/VetrinaDolci.WebAPI/Models/DolceInVendita.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace VetrinaDolci.WebAPI.Models
{
    public class DolceInVendita
    {
        public int Id { get; set; }
        public int Disponibilita { get; set; }
        public DateTime InVenditaDa { get; set; }
        public int? DolceId { get; set; }
        public Dolce Dolce { get; set; }
        [NotMapped]
        public double? Prezzo { get; set; }
    }
}

File: src/vetrina-dolci-client/src/app/models/dolce-in-vendita.ts
import { Dolce } from "./dolce";

export class DolceInVendita {
    id: number;
    disponibilita: number;
    inVenditaDa: Date;
    prezzo: number;
    img: string;
    
    dolceId: number;
    dolce: Dolce;
    
    constructor(data?: Partial<DolceInVendita>) {
      Object.assign(this, data);
    }
  }
  